lib/options-parser.js

```diff
@@ -44,6 +46,7 @@ function parseOptions(argsOriginal) {
     parsed.url = args[0];
     parsed.css = args[1];
 
+    // this is problematic...
```

Comment:
Hmm, so down the line we can simplify the tests too - no need for us either to start up a server.

Will remove this check now.